tests/ports/qemu/asm_test.py

```diff
@@ -1,4 +1,13 @@
-import frozen_asm
+import sys
+
+arch = getattr(sys.implementation, "_mpy", 0) >> 10
+if arch >= 3 and arch <= 8:
+    import frozen_asm_thumb as frozen_asm
+elif arch == 11:
+    import frozen_asm_rv32 as frozen_asm
+else:
+    raise ImportError
```

Comment:
Would it be simpler and equivalent to do:
```py
try:
    import frozen_asm_thumb as frozen_asm
except ImportError:
    try:
        import frozen_asm_rv32 as frozen_asm
    except ImportError:
        raise ImportError
```
?

Then you don't need to know about the different arch values.

Reply:
Makes sense.  This should be fixed in the latest iteration of this PR.